### PR TITLE
Chore correct words

### PR DESCRIPTION
**Pull request by Howie59**
**What type of PR is this?**
/kind cleanup

**Which issue(s) this PR fixes**:
- Correct words and format codes

```release-note
NONE
```

### CODE REVIEW

**Comment by Howie66** [comment]
> /assign @deads2k

/assign @deads2k

File: staging/src/k8s.io/client-go/transport/config.go [hunk 1]
```diff
@@ -132,8 +132,8 @@ type TLSConfig struct {
 	ServerName string // Override for the server name passed to the server for SNI and used to verify certificates.
 
 	CAData   []byte // Bytes of the PEM-encoded server trusted root certificates. Supercedes CAFile.
-	CertData []byte // Bytes of the PEM-encoded client certificate. Supercedes CertFile.
```
Comment (liggitt) [inline staging/src/k8s.io/client-go/transport/config.go:135]: revert this change, this is a valid spelling

File: staging/src/k8s.io/apimachinery/pkg/watch/watch.go [hunk 2]
```diff
@@ -276,7 +276,7 @@ func (f *RaceFreeFakeWatcher) Action(action EventType, obj runtime.Object) {
 	}
 }
 
-// ProxyWatcher lets you wrap your channel in watch Interface. Threadsafe.
```
Comment (liggitt) [inline staging/src/k8s.io/apimachinery/pkg/watch/watch.go:279]: Revert changes to "threadsafe"

File: staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go [hunk 3]
```diff
@@ -289,7 +289,7 @@ func IsInRange(value int, min int, max int) []string {
 	return []string{InclusiveRangeError(min, max)}
 }
 
-// Now in libcontainer UID/GID limits is 0 ~ 1<<31 - 1
```
Comment (liggitt) [inline staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go:292]: revert this change

**Review by liggitt — COMMENTED** [review]
The typo fixes in the comments look fine (except where noted). Please revert all the variable and file name changes, those all look reasonable as they were.

**Comment by Howie66** [comment]
> The typo fixes in the comments look fine (except where noted). Please revert all the variable and file name changes, those all look reasonable as they were.

ok, i have revert the changes you noted, thx

File: staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go [hunk 4]
```diff
@@ -43,13 +43,13 @@ type Aggregate interface {
 // this returns nil.
 // It will check if any of the element of input error list is nil, to avoid
 // nil pointer panic when call Error().
-func NewAggregate(errlist []error) Aggregate {
-	if len(errlist) == 0 {
```
Comment (liggitt) [inline staging/src/k8s.io/apimachinery/pkg/util/errors/errors.go:47]: changing variable names like this masks history for little benefit… the existing variable names in this PR are generally fine as they were. Let's limit this PR to correcting actual typos

**Comment by liggitt** [comment]
looks good, please squash to a single commit, then lgtm

**Comment by Howie66** [comment]
> looks good, please squash to a single commit, then lgtm

ok, i have finished the squash

**Comment by Howie66** [comment]
> /retest

/retest

**Comment by Howie66** [comment]
> /retest

/retest

**Comment by Howie66** [comment]
@deads2k 
it seems like my issue lack the v1.19, can u add on it?

**Comment by Howie66** [comment]
v1.19

**Comment by Howie66** [comment]
@caesarxuchao hi, it seems like my issue lack the `/milestone v1.19`, could u add on it?

**Comment by liggitt** [comment]
Non-release blocking changes like this will need to wait until master reopens for 1.20. We're currently finalizing the 1.19 release (targeting 8/25) and are in code freeze

**Comment by Howie66** [comment]
> Non-release blocking changes like this will need to wait until master reopens for 1.20. We're currently finalizing the 1.19 release (targeting 8/25) and are in code freeze

ok, thx

**Comment by Howie66** [comment]
@asalkeld @caesarxuchao @liggitt hi, it seems like v1.20 has opened, can this pr be merged?

**Comment by liggitt** [comment]
This will merge shortly, 1.20 merges are [reopening in phases](https://docs.google.com/document/d/1yI7o7R2bOB7QtkHUFMHMb5XObiJmzIAnJPuxhdAX-CA/edit)
